public/src/app/app.component.html

```diff
@@ -9,41 +9,52 @@
             <a target='_blank' href='https://github.com/keymanapp/keyman/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3A{{phase?.node.title}}+org%3Akeymanapp'>{{phase?.node.title}}</a>
             <span class="navbar-phase-dates">{{phaseStart}} &mdash; {{phaseEnd}}</span>
           </span>
-          <span class="navbar-new-issues"><a target='_blank' href="https://github.com/keymanapp/keyman/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone" class="label hide-if-zero {{errorClassIfNonZero(status?.github.data.repository.issuesWithNoMilestone.totalCount)}}">New issues: {{status?.github.data.repository.issuesWithNoMilestone.totalCount}}</a></span>
+          <span class="navbar-new-issues">
+            <a target='_blank' href="https://github.com/keymanapp/keyman/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone" class="label hide-if-zero {{errorClassIfNonZero(status?.github.data.repository.issuesWithNoMilestone.totalCount)}}">New issues: {{status?.github.data.repository.issuesWithNoMilestone.totalCount}}</a>
+          </span>
+          <span class="navbar-new-pulls">
+            <span *ngFor="let pull of unlabeledPulls">
+              <app-pull-request [class]="'failure'" [pull]="pull"></app-pull-request>
+              <!--
+                <a href="{{pull.pull.node.url}}" class="label {{pullClass(pull)}}" title="{{pull.pull.node.title}}: {{pull.state?.description}}" target="_blank">{{pull.pull.node.number}}</a>
+              -->
+            </span>
+          </span>
         </div>
     </div>
 </nav>
 
 <!-- <h1></h1> -->
 
 <div class="container-fluid">
-  <table class="table table-striped">
+  <table class="table table-striped table-bordered">
     <thead class='thead-dark'>
       <tr>
         <th id='th-platform'>Platform</th>
-        <th id='th-alpha' class='tier'>Alpha</th>
+        <th id='th-alpha' class='tier'>Stable</th>
         <th id='th-beta' class='tier'>Beta</th>
-        <th id='th-stable' class='tier'>Stable</th>
+        <th id='th-stable' class='tier'>Alpha</th>
```

Comment:
Why did these switch where they don't match the header id's?

Reply:
Because I switched the order of the columns but got the header switch wrong ;-)